TrackingTools/GsfTools/plugins/BuildFile.xml

```diff
@@ -1,3 +1,4 @@
+<flags   CXXFLAGS="-Ofast"/>
```

Comment:
I actually tried removing -Ofast, but it did not affect the trigger
results, i.e. the same discrepancy is there with and without it.

.A